### ISSUE TITLE
Tweet-embedding and recommendation sidebar are not working on infinite scroll blog posts

### ISSUE THREAD

**Issue by lostboyevsky** [jetlej/HE-6.0-Bugs#255]
If you go to any blog post and scroll down, down, down until you get to the next blog post, you will find that the recommendation sidebar is not working on any blog posts below the top one in the infinite scroll. I also noticed that tweets are not embedding correctly for any except the top blog post on infinite scroll --- the tweets just show up as if they are quotes within the <blockquote></blockquote> tags. For instance, the blockquote in this screenshot is supposed to be a tweet (also note the lack of a recommendation sidebar):

![screen shot 2016-02-18 at 3 37 05 pm](https://cloud.githubusercontent.com/assets/9164601/13161897/bdd56c56-d655-11e5-999c-295bd6498b4c.png)

**Comment by dpatil-gmcs** [comment]
@lostboyevsky @MSchirp : Yes, we too have observed this issue while we were figuring out a solution for #250...Seems like the code implemented just STRIPS OFF any Javascript and hence the issues in #250 and #255. We will have to OVERWRITE the already implemented Code (by HighExistence) to make it work.

**Comment by lostboyevsky** [comment]
I see. For the record, @dpatil-gmcs, this issue isn't particularly urgent, as we don't have that many articles with embedded tweets, so few visitors will see this problem.
